src/modules/search/components/AutocompleteSearch.js

```diff
@@ -78,24 +59,24 @@ export class AutocompleteSearch extends BaElement {
 			if (e.keyCode == 40) {
 				/*If the arrow DOWN key is pressed,
 				increase the currentFocus variable:*/
-				currentFocus++;
+				this._currentFocus++;
```

Comment:
To get it working properly, i have to lift this variable up to an "internal-state", otherwise i have no access from 
`this._clearCandidates`